File: src/backend/VoteBem/Entities/CertidaoCriminal.cs
namespace VoteBem.Entities
{
    public class CertidaoCriminal
    {
        public long IdCertidao { get; private set; }
        public long SqCandidato { get; private set; }
        public string NmArquivo { get; private set; } = null!;
        public string? DsCaminhoArquivo { get; private set; }
        public DateOnly? DtEmissao { get; private set; }
        public DateOnly? DtValidade { get; private set; }

        public Candidatura Candidatura { get; private set; } = null!;

        protected CertidaoCriminal() { }
    }
}

File: src/etl/models/certidao_criminal.py
from sqlalchemy import Column, BigInteger, String, ForeignKey, Date
from .base import Base


class CertidaoCriminal(Base):
    __tablename__ = "certidao_criminal"

    id_certidao = Column(BigInteger, primary_key=True, autoincrement=True)
    sq_candidato = Column(BigInteger, ForeignKey("candidatura.sq_candidato"), nullable=False)
    nm_arquivo = Column(String, nullable=False)
    ds_caminho_arquivo = Column(String)
    dt_emissao = Column(Date)
    dt_validade = Column(Date)
